manifests/application.pp

```diff
@@ -20,7 +20,6 @@
   Boolean $use_pulp_2_for_file = false,
   Boolean $use_pulp_2_for_docker = false,
   Boolean $use_pulp_2_for_yum = false,
```

Comment:
These can be dropped as well if there's no Pulp 2.

Reply:
I agree they ought to be dropped, just waiting for confirmation from @jlsherrill that they have been dropped code-wise inside Katello.

Reply:
The code suggests that if it's not present, it equals to false so should be safe to drop:
https://github.com/Katello/katello/blob/cf1bf09002252669c997b6317868582d2c35b465/app/models/katello/concerns/smart_proxy_extensions.rb#L149-L156

Reply:
yep +1 safe to drop